examples/appdir/src/app/page.tsx

```diff
@@ -1,13 +1,9 @@
 "use client";
 
-import { Inter } from "next/font/google";
-
 import { UploadButton, UploadDropzone } from "@uploadthing/react";
 
 import type { OurFileRouter } from "~/server/uploadthing";
 
-const inter = Inter({ subsets: ["latin"] });
```

Comment:
Pretty sure these are "magic" might need to ignore them

Reply:
how do you mean? we don't do the `inter.className` or `inter.variable` anywhere?

Reply:
I had a misunderstanding about how this was supposed to work